.github/packager/packager.sh

```
@@ -4,19 +4,30 @@
 # if any command returns a non-zero exit status (i.e., an error).
 set -e
 
+# Helper function for cross-platform 'sed -i'
 replace_in_place() {
-  if [[ "$OSTYPE" == "darwin"* ]]; then
+  if [ "$OSTYPE" = "darwin"* ]; then  # FIX: Changed [[...]] to [...] for sh compatibility
```

Shellcheck (0.11.0)

[error] [ .. ] can't match globs. Use [[ .. ]] or case statement.

(SC2081)